README.md

```diff
@@ -2,19 +2,25 @@
 
 # Complete the deputy report
 
-This app is the [Complete the deputy report][service] service. It provides an online reporting service used by deputies to submit their reports, and the private area for case managers to review submitted reports.
+This app is the [Complete the deputy report][service] service. It provides an online reporting service used by deputies
+to submit their reports, and the private area for case managers to review submitted reports.
```

Comment:
I think substituting private for admin here makes sense as that's the actual name of the service. Or maybe:

```suggestion
This app is the [Complete the deputy report][service] service. It provides an online reporting service that has a publicly available frontend used by deputies
to submit their reports, and a VPN restricted admin area for case managers to review submitted reports.
```